### PR TITLE
Add hungryhowies.com

### PR DESCRIPTION
**Pull request by Caboosey**

### CODE REVIEW

File: src/chrome/content/rules/hungryhowies.com.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="hungryhowies.com">
+        <target host="hungryhowies.com" />
+        <target host="www.hungryhowies.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/hungryhowies.com.xml]: Please also add the following targets:
franchising.hungryhowies.com
home.hungryhowies.com

Comment (Caboosey) [inline src/chrome/content/rules/hungryhowies.com.xml]: Added.

File: src/chrome/content/rules/hungryhowies.com.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        <target host="franchising.hungryhowies.com />
+        <target host="home.hungryhowies.com />
```
Comment (J0WI) [inline src/chrome/content/rules/hungryhowies.com.xml]: There is a missing `"` resulting in a syntax error.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**